scripts/install_halide_deps.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -e
+
+SOURCE="${BASH_SOURCE[0]}"
+REQUIREMENTS=`(dirname ${SOURCE})`/requirements.txt
+
+if [ ! -d "/usr/lib/llvm-7" ]; then
```

Comment:
I'm not sure if llvm-7 is required to run, since `lihHalide.a` ships with llvm-7. It it does, let me know and I will update the base docker image.

Reply:
The repo doesn't need llvm to be installed in the normal garnetflow build. I only need to to build libHalide.a (since I now use the GarnetFlow setup to build libHalide releases in the travis build for Halide-to-Hardware.

I moved this code to `install_and_build_halide.sh`, which is not used in the GarnetFlow build.